backend/src/main/java/org/example/backend/service/AssemblyAiService.java

```diff
@@ -0,0 +1,46 @@
+package org.example.backend.service;
+
+import org.example.backend.model.assemblyai.AssemblyAiResponse;
+import org.springframework.core.io.ByteArrayResource;
+import org.example.backend.model.assemblyai.FileUploadRequest;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+
+@Service
+public class AssemblyAiService {
+
+    // API Key für AssemblyAI
+    @Value("${assemblyai.api.key}")
+    private String assemblyApiKey;
+
+    private final WebClient webclient;
+
+    public AssemblyAiService(WebClient.Builder builder) {
+        this.webclient = builder.build();
+    }
```

Comment:
The `WebClient` instance in `AssemblyAiService` is built without specifying a base URL, possibly leading to redundant `.uri()` calls. Suggest fix is to set a base URL perhaps?

```suggestion
    public AssemblyAiService(WebClient.Builder builder) {
        this.webclient = builder.baseUrl("https://api.assemblyai.com/v2").build(); // (or whatever base url should be)
    }
```